docs/how-to-guides/new-user-guides/manage-clusters/clean-cluster-nodes.md

```diff
@@ -121,25 +121,41 @@ Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
 <Tabs>
 <TabItem value="RKE1">
 
-### Windows Nodes
+Before you run the following commands, first remove the node through the Rancher UI.
+
+To remove a node:
+
+1. Click **☰** and select **Cluster Management**.
+1. In the table of clusters, click the name of the cluster the node belongs to.
+1. In the **Machine Pool** tab, click the checkbox next to the node's state.
```

Comment:
in dashboard, this tab is just `machines`

Reply:
will the UI automatically fix 1. 1. 1. -> 1. 2. 3. ?

Reply:
@slickwarren I doublechecked, and my instructions lead me to this page, which has the tab labeled "Machine Pools". Are there different places where users can remove nodes? 

![image](https://github.com/rancher/rancher-docs/assets/19174201/b47cc37d-a9cb-45e6-863e-36e5d780931a)

Reply:
are you on 2.7? For custom clusters, the page looks like this for me on 2.7-head for an rke1 cluster
![Screen Shot 2023-06-02 at 11 39 54 AM](https://github.com/rancher/rancher-docs/assets/16691014/cf3b4986-81ca-43c6-9cc6-081d3cc3e032)

Reply:
Yep, 2.7.3. But I'm looking at the local cluster, not a downstream. Maybe that's it?

Reply:
ah, local cluster is an imported one. afaik, all windows clusters will be custom clusters

Reply:
With variable tab text, I'm tempted to use "in the first tab" instead of specifying "in the Machine Pools or Machines tab". Do you have any preferences on how to handle this?

Reply:
Honestly, I have no preference. first tab should work too